### PR TITLE
Make clear that bugs for packages aren't in this repo

### PR DESCRIPTION
**Pull request by puiterwijk**

### CODE REVIEW

**Comment by karlp** [comment]
:+1: helpful clarification text

File: fedoracommunity/templates/footer.mak [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <%def name="footer(package_name=None)">
         <div class="container_24">
           <div id="bottom" class="grid_24">
-            <p>Found a bug?  <a href="https://github.com/fedora-infra/fedora-packages/issues/new">File a ticket.</a>    Note:  There's some caching going on here.  If you expect something and don't see it, check back in 5 minutes.</p>
+            <p>Found a bug in this website?  <a href="https://github.com/fedora-infra/fedora-packages/issues/new">File a ticket.</a> For bugs in packages, click the "Open a New Bug" link on the packages Bugs page.   Note:  There's some caching going on here.  If you expect something and don't see it, check back in 5 minutes.</p>
```
Comment (pypingou) [inline fedoracommunity/templates/footer.mak]: in or on?

Comment (puiterwijk) [inline fedoracommunity/templates/footer.mak]: I have no idea what you mean? :-)

**Comment by pypingou** [comment]
:+1: for me

**Pull request merged**